### DISCUSSION TITLE
what are  names of the colorscheme highlight groups?

### DISCUSSION THREAD

**Discussion by johnmathews** [nvim-tree/nvim-tree.lua#1601] [Q&A]
Hi,

I'm having some weird issues where my colorscheme (monokai)[https://github.com/tanvirtin/monokai.nvim] breaks the colors in nvim-tree. All the text and icons appear white.

Id like to set the colors explicitly, where can I find a list of highlight groups?

**Comment by pleshevskiy** [comment]
https://github.com/kyazdani42/nvim-tree.lua/blob/master/lua/nvim-tree/colors.lua

**Comment by alex-courtis** [comment]
`:help nvim-tree-highlight`
